.config/CredScanSuppressions.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Should this file be here or in the ADO pipeline?

Reply:
It should be here. It's imported into ADO along with the other files in the Github Repository